posthog/temporal/workflows/snowflake_batch_export.py

```diff
@@ -174,6 +176,14 @@ async def insert_into_snowflake_activity(inputs: SnowflakeInsertInputs):
             )
             result = None
             local_results_file = tempfile.NamedTemporaryFile(suffix=".jsonl")
+            rows_in_file = 0
+
+            def flush_to_snowflake(lrf: tempfile._TemporaryFileWrapper, rows_in_file: int):
+                lrf.flush()
+                put_file_to_snowflake_table(cursor, lrf.name, inputs.table_name)
+                ROWS_EXPORTED.labels(destination="snowflake").inc(rows_in_file)
```

Comment:
I have a PR in draft to add BatchExportTemporaryFile to snowflake batch exports: https://github.com/PostHog/posthog/pull/18427.

This means that tracking can be the same as the other destinations instead of needing a separate `rows_in_file`. 

But we can merge this first, and I'll rebase 👍